compiler/ilgen/IlBuilder.cpp

```diff
@@ -516,16 +516,17 @@ IlBuilder::appendBlock(TR::Block *newBlock, bool addEdge)
    _currentBlock = newBlock;
    }
 
-
 void
-IlBuilder::AppendBuilder(TR::IlBuilder *builder)
+IlBuilder::AppendBuilder(TR::IlBuilder *builder, TR::Block *blockThrowsException)
```

Comment:
TR::Block is not part of the IlBuilder API, so this parameter can't be TR::Block...